src/Sulu/Bundle/WebsiteBundle/Sitemap/XmlSitemapRenderer.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Sitemap;
+
+use Sulu\Component\Webspace\Portal;
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+
+/**
+ * Render sitemap in xml-format.
+ */
+class XmlSitemapRenderer implements XmlSitemapRendererInterface
```

Comment:
I think I would have split that into a `Renderer` and a `Dumper`, to follow SRP.